src/robot/result/model.py

```diff
@@ -419,6 +419,44 @@ def doc(self):
         return ''
 
 
+@Body.register
+class Error(model.Error, StatusMixin, DeprecatedAttributesMixin):
+    __slots__ = ['status', 'starttime', 'endtime']
+    body_class = Body
+
+    def __init__(self, values=(), status='FAIL', starttime=None, endtime=None, parent=None):
+        super().__init__(values, parent)
+        self.status = status
+        self.starttime = starttime
+        self.endtime = endtime
+        self.body = None
+
+    @setter
+    def body(self, body):
+        """Child keywords and messages as a :class:`~.Body` object.
+
+        Typically empty. Only contains something if running BREAK has failed
+        due to a syntax error or listeners have logged messages or executed
+        keywords.
```

Comment:
`BREAK` reference is invalid here. Also, isn't there typically a message so the body isn't typically empty? There aren't "child keywords", though.